src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -197,32 +175,12 @@ def sample_simulants_from_standard_households(
 
         return new_simulants
 
-    def sample_simulants_from_group_quarters(
+    def initialize_group_quarters(
         self,
-        target_number_sims: int,
-        acs_households: pd.DataFrame,
         acs_persons: pd.DataFrame,
         pop_data: SimulantData,
     ) -> pd.DataFrame:
-        # group quarters each house one person per census_household_id
-        # they have NA household weights, but appropriately weighted person weights.
-        chosen_units_index = vectorized_choice(
-            options=acs_households.index,
-            n_to_choose=target_number_sims,
-            randomness_stream=self.randomness,
-            weights=acs_households["person_weight"],
-        )
-        chosen_units = acs_households.loc[chosen_units_index]
-
-        # get simulants per GQ unit
-        chosen_persons = pd.merge(
-            chosen_units,
-            acs_persons[metadata.PERSONS_COLUMNS_TO_INITIALIZE],
-            on="census_household_id",
-            how="left",
-        )
-
-        new_simulants = self.initialize_new_simulants_from_acs(chosen_persons, pop_data)
+        new_simulants = self.initialize_new_simulants_from_acs(acs_persons, pop_data)
         new_simulants["age"] = self.perturb_individual_age(new_simulants)
```

Comment:
Is there a reason we're calling `perturb_individual_age` two separate times rather than just once right before we update the pop view?

Reply:
If I'm understanding the question correctly, yes. When initializing the population, the non-GQ population uses `self.perturb_household_age` which makes a single shift at the household level. The GQ population is individually perturbed (`self.perturb_individual_age`).

Also, note that guardians depend on age. So initializing link columns is another step that must be done between age perturbation and updating the pop view.

Reply:
Thanks for that. I misread this and thought they were both the same function call, but one is `perturb_household_age` and the other is `perturb_individual_age`. Carry on.